runtime/src/main/java/org/corfudb/runtime/view/AddressSpaceView.java

```diff
@@ -257,12 +268,13 @@ ILogData read(long address) {
                 // load the same value), but currently a redundant RPC
                 // is much cheaper than the cost of a NoRollBackException, therefore
                 // this trade-off is reasonable
-                final ILogData loadedVal = fetch(address);
+                final ILogData loadedVal = fetch(address, ignoreTrimmed);
```

Comment:
If ignoreTrimmed is true, then this will pollute the cache with trimmed entries, right?

Reply:
yes, you're right I'll remove the trimmed entries!